pyproject.toml

```diff
@@ -28,6 +29,16 @@ safety = "^2.3.1"
 sphinx = "4.3.2"
 sphinx-rtd-theme = "^1.0.0"
 
+
+[tool.poetry.group.dev]
+optional=true
+
+[tool.poetry.group.dev.dependencies]
+python-lsp-server = {extras = ["rope"], version = "^1.6.0"}
```

Comment:
Hmm, does anyone else use these dependencies? I wonder how the team feels about adding such specific development tooling to our common pyproject?

Reply:
I was also conflicted about this, the lsp tooling I use was deeply confused by poetry and this seemed to be the workaround -- do you know if there's a less obnoxious way for me to include those as dev dependencies? I could also just not commit those changes to pyproject.toml on my end, but that feels like asking for trouble